spec/50-privacy.md

```diff
@@ -6,11 +6,26 @@ Trace vendors MUST assess the risk of header abuse. This section provides some c
 
 ## Privacy of traceparent field
 
-The `traceparent` field has a predefined set of values. These values are randomly-generated numbers. If a random number generator has any logic of using user-identifiable information like IP address - this information may be exposed. Random number generators MUST NOT rely on any information that can potentially be user-identifiable.
-
-Another privacy risk of the `traceparent` field is an ability to correlate calls made as part of a single transaction. A downstream service may track and correlate two or more calls made in a single transaction and make assumptions about the identity of the caller of one call base on information in another call. Service initiating calls MAY choose to restart a trace while making calls that might identify caller in the downstream service.
-
-Note, both privacy concerns of `traceparent` field are theoretical rather than practical.
+The `traceparent` field has a predefined set of values. These values are randomly-generated numbers. If a random number generator has any logic of using user identifiable information like IP address - this information may be exposed. Random number generators MUST NOT rely on any information that can potentially be user identifiable.
+
+Another privacy risk of the `traceparent` field is an ability to correlate calls made as part of a single transaction. A downstream service may track and correlate two or more calls made in a single transaction and make assumptions about the identity of the caller of one call base on information in another call.
+
+Note, both privacy concerns of `traceparent` field are theoretical rather than
+practical. Some services initiating or receiving a call MAY choose to restart a
+`traceparent` field to eliminate those risks completely. It is recommended to
+find a way to minimize the number of <a>distributed trace</a> restarts to promote
```

Comment:
```suggestion
find a way to minimize the number of distributed trace restarts to promote
```

Reply:
this is needed to create a link to glossary